src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -15,13 +15,10 @@ class GradleBuilder extends AbstractBuilder {
   // https://issues.jenkins.io/browse/JENKINS-47355 means a weird super class issue
   def localSteps
 
-  def securitytest
-
   GradleBuilder(steps, product) {
     super(steps)
     this.product = product
     this.localSteps = steps
-    this.securitytest = new SecurityScan(this.steps)
```

Comment:
removed as it is a duplicate call. The call already exists in the super class `AbstractBuilder.groovy` and is implicitly called on the line 19 `super(steps)` f.y.i.  @reespozzi